src/box/lua/config/applier/box_cfg.lua

```diff
@@ -351,6 +351,7 @@ local function apply(config)
     box_cfg.cluster_name = names.group_name
     box_cfg.replicaset_name = names.replicaset_name
     box_cfg.instance_name = names.instance_name
+    box_cfg.bootstrap_leader = configdata:bootstrap_leader()
```

Comment:
Nit: it is fused with the block regarding names that at least makes the comment to the block incomplete:

> -- Add instance, replicaset and group (cluster) names.

I would move it into its own block.

Reply:
Fixed.